### ISSUE TITLE
Cmd-T is a core keybinding already

### ISSUE THREAD

**Issue by gampleman** [mbuscemi/elm-test-runner#8]
Cmd-T is already used in Atom to bring up the file selector. I't would be nice to have something that doesn't conflict with the defaults.

**Comment by mbuscemi** [comment]
It appears there's no Atom documentation for the key bindings that they'd like to reserve for the core editor, and so I've been looking through the Keybindings panel under Atom's Settings.

I'd like to keep the test run command simple, and it looks like both Ctrl-T and Cmd-T are taken, so that leaves Alt-T.

Toggling the panel open and closed can remain Cmd-Alt-T. I've been meaning to normalize "toggle auto-run" command with the other option settings, too, making it Ctrl-Alt-R. I don't see any other conflicts.

So, to summarize:

Run Tests: Change from Cmd-T to Alt-T
Toggle Test Pane: Alt-Cmd-T

Toggle Auto-Run on Save: Change from Ctrl-R to Ctrl-Alt-R
Toggle Auto-Navigate: Ctrl-Alt-N
Toggle Elm Verify Examples: Ctrl-Alt-V

What do you think of this proposal?

I'll roll these changes into 0.3.0 (#4) unless there is some urgent need for a patch.

**Comment by gampleman** [comment]
Sounds good to me.

**Comment by mbuscemi** [comment]
Fixed in 0.2.1.
